docs/cleaning-up.md

```diff
@@ -0,0 +1,62 @@
+# Cleaning Up
+
+Done with your Psoxy deployment?
+
+Terraform makes it easy to clean up when you're through with Psoxy, of you wish to rebuild
+everything from scratch.
+
+First, a few caveats:
+  - this will NOT undo any changes outside of Terraform, even those we instructed you to perform
+    via `TODO - ` files that Terraform may have generated.
+  - be careful with anything you created outside of Terraform and later imported into Terraform,
+    such as GCP project / AWS account themselves. If you DON'T want to destroy these, do
```

Comment:
Is that the reason about not to use `terraform apply -destroy` instead of ? This is what I had in https://github.com/Worklytics/psoxy/tree/main/infra/examples/gcp-hris#deployment, in case you want to remove that section too